2-extra/2-water-bottle.js

```diff
@@ -18,18 +18,29 @@ You have to implement the missing features according to the specification.
 let bottle = {
   volume: 0,
   fillUp: function () {
+    this.volume = 100;
     // calling this function should completely fill your bottle (volume = 100);
   },
   pour: function () {
+    if (this.volume < 100){
+      this.volume += 10;
+    }
+    
     // calling this function should increase your bottle volume by 10 units;
   },
   drink: function () {
+    if(this.volume >= 10){
+      this.volume -= 10;
+    }
+    
     // calling this function should decrease your bottle volume by 10 units;
   },
   isFull: function () {
+    return this.volume == 100;
```

Comment:
Try to use `===` instead of `==` here.
Other than that - everything else in this exercise looks good 👍